src/plugins/hetero/executable_network.cpp

```diff
@@ -488,16 +489,23 @@ HeteroExecutableNetwork::HeteroExecutableNetwork(std::istream& heteroModel,
     pugi::xml_node outputsNode = heteroNode.child("outputs");
     FOREACH_CHILD (outputNode, outputsNode, "output") { networkOutputs.insert(GetStrAttr(outputNode, "name")); }
 
+    auto config = user_config;
     auto heteroConfigsNode = heteroNode.child("hetero_config");
     FOREACH_CHILD (heteroConfigNode, heteroConfigsNode, "config") {
-        _hetero_config.emplace(GetStrAttr(heteroConfigNode, "key"), GetStrAttr(heteroConfigNode, "value"));
+        config.emplace(GetStrAttr(heteroConfigNode, "key"), GetStrAttr(heteroConfigNode, "value"));
     }
 
     auto deviceConfigsNode = heteroNode.child("device_config");
     FOREACH_CHILD (deviceConfigNode, deviceConfigsNode, "config") {
-        _device_config.emplace(GetStrAttr(deviceConfigNode, "key"), GetStrAttr(deviceConfigNode, "value"));
+        config.emplace(GetStrAttr(deviceConfigNode, "key"), GetStrAttr(deviceConfigNode, "value"));
     }
 
+    // Split `user_config` (via `config` as updated from imported model)
+    // on `device_config` (with properties for underlying devices)
+    // and `_cfg` (hetero config with properties for hetero plugin only)
+    _cfg = ov::hetero::Configuration(config, plugin->m_cfg);
+    auto device_config = config;
```

Comment:
Looks like you don't change `device_config` so I think you can use `config` instead of creation new local variable

Reply:
Yes, will be removed